terraformutils/utils.go

```diff
@@ -106,6 +108,8 @@ func RefreshResources(resources []*Resource, provider *providerwrapper.ProviderW
 	for _, resourceGroup := range slowProcessingResources {
 		for i := range resourceGroup {
 			r := resourceGroup[i]
+			log.Printf("slow processing Instance state %v  \n", r.InstanceState)
```

Comment:
please delete, no need here

Reply:
@sergeylanzman done in c07585a114788b9a428a7d601d501196d4306e73